src/Backend/Modules/Analytics/GoogleClient/Connector.php

```diff
@@ -176,21 +176,22 @@ private function getData($startDate, $endDate)
     {
         $dateRange = $startDate . '-' . $endDate;
 
-        if (!$this->cache->isCached($dateRange)) {
-            $this->cache->cache(
-                $dateRange,
-                json_encode(
-                    array(
-                        'metrics' => $this->getMetrics($startDate, $endDate),
-                        'visitGraphData' => $this->collectVisitGraphData($startDate, $endDate),
-                        'pageViews' => $this->collectMostVisitedPagesData($startDate, $endDate),
-                        'sourceGraphData' => $this->collectSourceGraphData($startDate, $endDate),
-                    )
-                )
-            );
+        $item = $this->cache->getItem('analytics-'.$dateRange);
```

Comment:
It only does it if you use the symfony standards. Using --level=psr2, the
spaces don't get removed.

Op donderdag 24 september 2015 heeft Matthias Mullie <
notifications@github.com> het volgende geschreven:

> In src/Backend/Modules/Analytics/GoogleClient/Connector.php
> <https://github.com/forkcms/forkcms/pull/1301#discussion_r40339247>:
>
> > @@ -176,21 +176,22 @@ private function getData($startDate, $endDate)
> >      {
> >          $dateRange = $startDate . '-' . $endDate;
> >
> > -        if (!$this->cache->isCached($dateRange)) {
> > -            $this->cache->cache(
> > -                $dateRange,
> > -                json_encode(
> > -                    array(
> > -                        'metrics' => $this->getMetrics($startDate, $endDate),
> > -                        'visitGraphData' => $this->collectVisitGraphData($startDate, $endDate),
> > -                        'pageViews' => $this->collectMostVisitedPagesData($startDate, $endDate),
> > -                        'sourceGraphData' => $this->collectSourceGraphData($startDate, $endDate),
> > -                    )
> > -                )
> > -            );
> > +        $item = $this->cache->getItem('analytics-'.$dateRange);
>
> FYI: php-cs-fixer removes spaces around concat, so I tend to follow that
> myself. But yes, most important is to keep it consistent throughout the
> project - I'll update it!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/forkcms/forkcms/pull/1301/files#r40339247>.
>